web/src/components/AssertionCard/AssertionCard.tsx

```diff
@@ -57,10 +66,23 @@ const AssertionCard: React.FC<TAssertionCardProps> = ({
     >
       <S.Header onClick={handleOnClick}>
         <div>
-          <AssertionCardSelectorList selectorList={selectorList} pseudoSelector={pseudoSelector} />
+          <AssertionCardSelectorList
+            originalSelector={originalSelector}
+            selectorList={selectorList}
+            pseudoSelector={pseudoSelector}
+          />
         </div>
         <S.ActionsContainer>
-          {isDraft && <S.StatusTag>draft</S.StatusTag>}
+          {isDraft && (
+            <>
+              <S.StatusTag>draft</S.StatusTag>
+              <Tooltip color="white" title="Revert Assertion">
```

Comment:
It is kind of weird that both the tooltip an the actual message have the same text